client/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "comments-client",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --mode=development",
+    "build-prod": "webpack --mode=production",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "prettier": "prettier --write 'src/**/*scss'"
+  },
+  "keywords": [],
+  "author": "Lisa Halvorsen",
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "^7.11.1",
+    "@fullhuman/postcss-purgecss": "^2.3.0",
```

Comment:
Since we agreed to go for vanila js/ts and not any stylesheet frameworks  we can also remove this purgecss dependencies.